packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -673,7 +673,11 @@ export class ApiGateway {
     }
   }
 
-  protected async defaultCheckAuth(req, auth) {
+  protected async defaultCheckAuth(req: Request, auth: string|undefined, schema: string) {
```

Comment:
should we default to the `bearer` schema?

Reply:
It's a default implementation for `checkAuth` middleware. I pass `bearer` as default value inside `extractAuthorizationHeaderWithSchema`.